src/Service/ZdsToZgwService.php

```diff
@@ -696,6 +696,7 @@ public function documentActionHandler(array $data, array $config): array
             // Set the zaakinformatieobject to the data.documents array.
             $this->data['documents'][] = $zaakInformatieObject->toArray();
 
+            $response = $this->createResponse($this->mappingService->mapping($mappingOut, $zaakInformatieObject->toArray()), 200);
             // Set the response with the mapped zaakinformatieobject to Bv03.
             $this->data['response'] = $this->createResponse($this->mappingService->mapping($mappingOut, $zaakInformatieObject->toArray()), 200);
```

Comment:
is this correct? feels weird to do this twice?
maybe do ['response'] = $response

Reply:
I'm not sure why this is done twice, I can look into it, but I don't have all information.

Reply:
Did you test this? looks like a duplicate in code.
Can you estimate if this might break things or not? if it might, can you look into this?

Reply:
I tested it, and it doesn't seem to break anything.